import_openssl.sh

```diff
@@ -256,16 +275,13 @@ function generate_build_config_headers() {
     PERL=/usr/bin/perl run_verbose ./Configure $CONFIGURE_ARGS ${!configure_args_bits} ${!configure_args_stat}
   fi
 
-  make include/openssl/configuration.h include/openssl/opensslv.h
-  make include/crypto/bn_conf.h include/openssl/lhash.h include/crypto/dso_conf.h
-  make providers/common/include/prov/der_ec.h providers/common/include/prov/der_ecx.h providers/common/include/prov/der_sm2.h providers/common/include/prov/der_dsa.h providers/common/include/prov/der_rsa.h providers/common/include/prov/der_digests.h providers/common/include/prov/der_wrap.h
-  make providers/common/der/der_digests_gen.c providers/common/der/der_ecx_gen.c providers/common/der/der_ec_gen.c providers/common/der/der_dsa_gen.c  providers/common/der/der_rsa_gen.c providers/common/der/der_sm2_gen.c providers/common/der/der_wrap_gen.c
-  make include/openssl/asn1.h include/openssl/asn1t.h include/openssl/bio.h include/openssl/cmp.h include/openssl/cms.h include/openssl/conf.h include/openssl/configuration.h include/openssl/crmf.h include/openssl/crypto.h include/openssl/ct.h include/openssl/err.h include/openssl/ess.h
-  make include/openssl/fipskey.h include/openssl/ocsp.h include/openssl/opensslv.h include/openssl/pkcs12.h include/openssl/pkcs7.h include/openssl/safestack.h include/openssl/srp.h include/openssl/ssl.h include/openssl/ui.h include/openssl/x509.h include/openssl/x509_vfy.h include/openssl/x509v3.h
-  make include/openssl/core_names.h include/internal/param_names.h crypto/params_idx.c
-  make include/openssl/comp.h include/openssl/x509_acert.h
+  # Refactored to this, because it search for all existing *.in files to transform it in *.h and *.c files.
+  # There is no need to search and declare them manually anymore, now it's search automatically.
+  local list_existing_in_files=$(find crypto include providers -name '*.in' | awk '{gsub(/\.in/, ""); print}' | sort -u)
+  echo -e "Founded *.in files are:\n$list_existing_in_files\n"
+  for input_in_files in "${list_existing_in_files[@]}"; do make $input_in_files; done
 
-  rm -f apps/CA.pl.bak openssl/opensslconf.h.bak
+  #rm -f apps/CA.pl.bak openssl/opensslconf.h.bak # Comment out, because these files are not exist anymore.
```

Comment:
Just remove it completely instead of commenting it out. The reason of change belong in the commit message.